backend/spellchecker/server/server.py

```diff
@@ -58,5 +62,41 @@ def serve(port: str, max_workers: int = 2):
     except KeyboardInterrupt:
         server.stop(0)
 
-if __name__ == '__main__':
-    serve(50051)
+if __name__ == "__main__":
+    args_parser = argparse.ArgumentParser(description="Spellchecker gRPC server.")
+    args_parser.add_argument("-w", "--workers",
+                             action="store",
+                             type=int,
+                             default=2,
+                             metavar="NUM",
+                             help="size of thread pool")
+
+    required = args_parser.add_argument_group("required")
+    required.add_argument("-p", "--port",
+                          required=True,
+                          action="store",
+                          type=int,
+                          metavar="NUM",
+                          help="port which server would be listening")
+    required.add_argument("-l", "--libparser",
+                          required=True,
+                          action="store",
+                          type=str,
+                          metavar="PATH",
+                          help="path to libparser shared library")
+    required.add_argument("-d", "--dir",
+                          required=True,
+                          action="store",
+                          type=str,
+                          metavar="PATH",
+                          help="path to .dic and .aff files")
+    required.add_argument("-L", "--language",
```

Comment:
which means that we're restricted to one language, since this argument is single-valued.

Why don't you initialize hunspell instance with the languagw requested bythe client on demand?